packages/welcome/lib/changelog-view.js

```diff
@@ -50,22 +50,19 @@ export default class ChangeLogView {
             <p>Feel free to read our <a href="https://github.com/pulsar-edit/pulsar/blob/master/CHANGELOG.md">Full Change Log</a>.</p>
             <ul>
               <li>
-                Added a new "UI" API to `atom`, accessible via `atom.ui`. This exposes a `markdown` object, allowing community packages to offload Markdown handling to the core editor.
+                Added the ability for a user to indicate when to automatically show or hide the wrap-guide; between always, when soft wrap is enabled, or when soft wrap at preferred line length is enabled.
```

Comment:
```suggestion
                Added the ability for a user to indicate when to automatically show or hide the wrap-guide; between always, when soft wrap is enabled, or when soft wrap at preferred line length is enabled.
```
                Added options for a user to control when to automatically show or hide the wrap-guide; "Always", "When soft wrap is enabled", and "When soft wrap at preferred line length is enabled".